packages/app/src/elements/item-view.ts

```diff
@@ -556,9 +575,7 @@ export class ItemView extends Routing(StateMixin(LitElement)) {
         }
     }
 
-    private async _attachFile() {
-        const file = this._fileInput.files![0];
-        this._fileInput.value = "";
+    private async _addFileAttachment(file: File) {
```

Comment:
It looks weird here, but I made this piece reusable, to use when clicking/selecting and when drag+dropping.